opencti-platform/opencti-front/src/private/components/analysis/malware_analyses/MalwareAnalysisEditionOverview.tsx

```diff
@@ -0,0 +1,401 @@
+        }
+    }
+`;
+
+export const malwareAnalysisEditionOverviewFragment = graphql`
+    fragment MalwareAnalysisEditionOverview_malwareAnalysis on MalwareAnalysis {
+        id
+        product
+        result_name
+        version
+        submitted
+        result
+        confidence
+        createdBy {
+            ... on Identity {
+                id
+                name
+                entity_type
+            }
+        }
+        objectMarking {
+            edges {
+                node {
+                    id
+                    definition_type
+                    definition
+                    x_opencti_order
+                    x_opencti_color
+                }
+            }
+        }
+        status {
+            id
+            order
+            template {
+                name
+                color
+            }
+        }
+        workflowEnabled
+    }
+`;
+interface MalwareAnalysisEditionOverviewProps {
+  malwareAnalysisRef: MalwareAnalysisEditionOverview_malwareAnalysis$key,
+  context: readonly ({
+    readonly focusOn: string | null;
+    readonly name: string;
+  } | null)[] | null
+  enableReferences?: boolean
+  handleClose: () => void
+}
+
+interface MalwareAnalysisEditionFormValues {
+  message?: string
+  references?: Option[]
+  createdBy?: Option
+  x_opencti_workflow_id: Option
+  objectMarking?: Option[]
+}
+
+// eslint-disable-next-line max-len
+const MalwareAnalysisEditionOverview: FunctionComponent<MalwareAnalysisEditionOverviewProps> = ({ malwareAnalysisRef, context, enableReferences = false, handleClose }) => {
+  const { t } = useFormatter();
+  const basicShape = {
+    result_name: Yup.string().required(t('This field is required')),
+    product: Yup.string().required(t('This field is required')),
+    submitted: Yup.date().typeError(t('The value must be a datetime (yyyy-MM-dd hh:mm (a|p)m)')),
+    confidence: Yup.number(),
+    result: Yup.string().nullable(),
+    x_opencti_workflow_id: Yup.object(),
+  };
+  const malwareAnalysisValidator = useSchemaCreationValidation('Malware-Analysis', basicShape);
+
+  const malwareAnalysis = useFragment(malwareAnalysisEditionOverviewFragment, malwareAnalysisRef);
+
+  const createdBy = convertCreatedBy(malwareAnalysis);
+  const objectMarking = convertMarkings(malwareAnalysis);
+  const status = convertStatus(t, malwareAnalysis);
+
+  const [commitRelationAdd] = useMutation<MalwareAnalysisEditionOverviewRelationAddMutation>(malwareAnalysisMutationRelationAdd);
+  const [commitRelationDelete] = useMutation<MalwareAnalysisEditionOverviewRelationDeleteMutation>(malwareAnalysisMutationRelationDelete);
+  const [commitFieldPatch] = useMutation<MalwareAnalysisEditionOverviewFieldPatchMutation>(malwareAnalysisMutationFieldPatch);
+  const [commitEditionFocus] = useMutation<MalwareAnalysisEditionOverviewFocusMutation>(malwareAnalysisEditionOverviewFocus);
+
+  const handleChangeCreatedBy = (_: string, value: Option) => {
```

Comment:
Why do you create a method that has a first parameter if you don't use it ?

Reply:
because what I need is the second parameter. The first one is name. I put instead (name: string, value: Option)